### ISSUE TITLE
Accessibility: Page does not reflow, elements inaccessible on small screens (initial view)

### ISSUE THREAD

**Issue by nedredmond** [inklewriter/freeinklewriter#77]
The app is not usable on small screens (see also #28). This makes it unusable on phones and tablets and some small chromebook-like devices. While inconvenient, this is also an ADA violation because it makes it inaccessible to users who need to zoom for low vision reasons.

## Recommended fix:
This is the big one. The site needs a fundamental redesign to either flow reasonably into smaller viewports or switch to alternate layouts (and entirely different workflows) for smaller screens. I would recommend the former to keep the maintenance burden down: use CSS to reflow areas of the page with CSS Grid and/or flexboxes; collapse the navigation and action menus into a hamburger menu; things like that.

May require a mobile-first redesign, and work backwards to desktop.

## ADA Violations:
- https://www.w3.org/WAI/WCAG22/Understanding/reflow.html

### Navigation tabs (write | read | etc) disappear at 794px width

<img width="797" height="640" alt="Image" src="https://github.com/user-attachments/assets/74688196-d1d0-4165-bd65-8459ec4addd9" />

### WCAG Reflow "400% zoom"
<img width="770" height="351" alt="Image" src="https://github.com/user-attachments/assets/e972204f-1c90-45c7-bc10-369d17f4ea24" />
